.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Chemprop - CPU",
+    "build": {
+        "context": "..",
+        "dockerfile": "../Dockerfile"
+    },
+    "settings": {
```

Comment:
I'm getting the following error:

    Property settings is not allowed.

    Use 'customizations/vscode/settings' instead(2)

    Machine specific settings that should be copied into the container. These are only copied when connecting to the container for the first time, rebuilding the container then triggers it again.

Can we fix this?

Reply:
n.b., this is on hover inside the dev container.

Reply:
what do you mean by "on hover"? I haven't seen that error yet.